resources/pool.rb

```diff
@@ -54,6 +54,7 @@
 attribute :disallow_rotation_on_config_change, kind_of: [TrueClass, FalseClass], default: false
 attribute :disallow_overlapping_rotation, kind_of: [TrueClass, FalseClass], default: false
 attribute :recycle_schedule_clear, kind_of: [TrueClass, FalseClass], default: false
+attribute :log_event_on_recycle, kind_of: String, default: nil
```

Comment:
This breaks current functionality, just make the default value be the default for IIS as all other attributes are `'Time, Requests, Schedule, Memory, IsapiUnhealthy, OnDemand, ConfigChange, PrivateMemory'`